src/services/symbols/EmojiService.js

```diff
@@ -0,0 +1,44 @@
+import emoji from 'node-emoji';
+
+
+export default class EmojiService {
```

Comment:
Maybe [actually use inheritance here](https://medium.com/ecmascript-2015/es6-classes-and-inheritance-607804080906).